### ISSUE TITLE
SKR 1.3 + TMC5160 CNC FAN always works

### ISSUE THREAD

**Issue by superkefir** [bigtreetech/BIGTREETECH-SKR-V1.3#260]
![IMG_0987 — копия](https://user-images.githubusercontent.com/47144059/76435262-d4ea7b80-63c7-11ea-815c-9816a3bfc6c1.JPG)
Hello,i have one problem. I install SKR 1.3 + TMC5160 v1.2 instead of MKS_GEN_L+2100 
But after configuration files when i turn on the printer it always burn LED signal near the CNC FAN out and always works fan of the layer. 
So when i connect it to my PC and send M107 or M106 S0 nothing changes. From TFT the same one.
I thought that it is the problem of binding P2_03, so i add #define FAN_PIN to pins_BTT_SKR_V1_3.h but it deosn\t help. After glashing firmware the fan starts always. How can i make it works properly?

[Marlin.zip](https://github.com/bigtreetech/BIGTREETECH-SKR-V1.3/files/4319149/Marlin.zip)

**Comment by superkefir** [comment]
Little update. I change tha motherboard to new one to test if the problem in mosfet.
So the situation is the same. When the DC24V fan connected it is turned off, when it's disconnected the led is turning off. I take multimeter to check voltage and change power of this pin. It was changed succesfully from 0V to 24V. So i thought there is a problem in a fan. I take another one 24V and connect it. It starts too too full power with power off to this pin. What i make wrong? I checked two 5015(24V) fans, one 4020, and two new 5015 (24v) from the box. So the situation is the same. I connect it to the motherboard without and cable. So the pin works and shows voltage only until i connect the fan to this pin.
